stable/opa/values.yaml

```diff
@@ -29,7 +29,7 @@ admissionControllerNamespaceSelector:
   matchExpressions:
     - {key: openpolicyagent.org/webhook, operator: NotIn, values: [ignore]}
 
-# SideEffectClass for the webhook, setting to None enables dry-run
+# SideEffectClass for the webhook, setting to None enables dry-run, if you want to disable the admissionControllerSideEffect option just comment the line below
 admissionControllerSideEffect: Unknown
```

Comment:
What's the mean of this? The default value is `Unknown `? , Is there need some more explains?

Reply:
The field is not available before Kubernetes v1.12 so validation with older versions is generating errors. Including this detail (with the version) in the comment would be nice.